### ISSUE TITLE
NPM Support?

### ISSUE THREAD

**Issue by jrasanen** [tenex/rails-assets#42]
Hi!

Any plans adding NPM support for proxying packages?

And thanks for continuing maintaining rails-assets.

**Comment by joshjordan** [comment]
Yes, we are very interested in supporting NPM. There are a number of challenges to solve there and we're not quite ready to begin development on NPM support yet, but yes, this is absolutely on our roadmap. Let's keep this issue open for tracking and +1's.

**Comment by fnando** [comment]
FYI, I'm working on a gem that converts NPM packages into Rubygems. It's working right now, including resolving/generating dependency packages as well. I was thinking about creating a rails-assets like for NPM because a) I think bower support should be dropped as NPM clearly won this fight and b) I didn't know that Tenex took over it. 

Anyway, here's a like: https://github.com/fnando/ene (check the examples directory)

**Comment by joshjordan** [comment]
Nando, that is awesome. Would love to chat with you a little bit more about
ene. Maybe theres an opportunity for us to work together! Want to shoot me
an email at josh.jordan@gmail.com ?
